architecture/standards/0006-use-of-provider-apis-in-gradle.md

```diff
@@ -39,6 +42,8 @@ class Example2 {
 ```
 This is unnecessary because this value is not calculated and cannot be changed.
 
+TODO: What should be done instead here?
```

Comment:
TODO

Reply:
For properties exposed as API, it may be just a matter of current implementation choice that the value is not changeable, so it could make sense to expose a non-calculated value as a lazy property, IMHO.